### PR TITLE
V1.4.1 Platform_events unit test

### PR DESCRIPTION
**Pull request by supritat**
### Requirements

* Filling out the template is required. Any pull request that does not include enough information to be reviewed in a timely manner may be closed at the maintainers' discretion.
* All new code requires tests to ensure against regressions

### Description of the Change
Included unit test for platform events

### CODE REVIEW

File: platform_events/index.js [hunk 1]
```diff
@@ -38,10 +38,11 @@ module.exports.handler = (event, context, cb) => {
 	const dynamodb = new AWS.DynamoDB();
 	const kinesis = new AWS.Kinesis();
 
+
 	try {
 		//GET Handler
 		if (event !== undefined && event.method !== undefined && event.method === 'GET') {
-
+			logger.info("GET Handler")
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: Removed log

File: platform_events/index.js [hunk 2]
```diff
@@ -133,13 +131,14 @@ module.exports.handler = (event, context, cb) => {
 							});
 							events.push(event);
 						});
-
 						if (results.get_events.LastEvaluatedKey !== undefined || results.get_events.LastEvaluatedKey !== "") {
+							logger.verbose('Get success')
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: removed log

File: platform_events/index.js [hunk 3]
```diff
@@ -133,13 +131,14 @@ module.exports.handler = (event, context, cb) => {
...
 							cb(null, responseObj({
 									"events": events,
 									"last_evaluated_key": results.get_events.LastEvaluatedKey
 								}, event.query));
 						} else {
+							logger.verbose('Get success')
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: removed log

File: platform_events/index.js [hunk 4]
```diff
@@ -154,6 +153,7 @@ module.exports.handler = (event, context, cb) => {
 
 		//POST Handler
 		if (event !== undefined && event.method !== undefined && event.method === 'POST') {
+			logger.info("POST Handler")
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: removed log

File: platform_events/index.js [hunk 5]
```diff
@@ -323,7 +319,7 @@ module.exports.handler = (event, context, cb) => {
 					}
 				},
 				store_context: ['validate_event_type', 'validate_event_name', 'validate_event_handler', 'validate_event_status', 'validate_timestamp', function (results, callback) {
-						var timestamp = moment().utc().format('YYYY-MM-DDTHH:mm:ss:SSS');
+					var timestamp = moment().utc().format('YYYY-MM-DDTHH:mm:ss:SSS');
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: done

File: platform_events/index.js [hunk 6]
```diff
@@ -389,7 +385,7 @@ module.exports.handler = (event, context, cb) => {
 						};
 						kinesis.putRecord(stream_params, function(err, data) {
 						  if (err) {
-
+							  logger.error('kinesis error');
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: error is logged using JSON.stringify

File: platform_events/index.js [hunk 7]
```diff
@@ -410,11 +406,13 @@ module.exports.handler = (event, context, cb) => {
 					if (err.code !== undefined && err.code === 500) {
 						cb(JSON.stringify(errorHandler.throwInternalServerError("An internal error occured. message: " + err.message)));
 					} else {
+						logger.error(err);
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: error is logged using JSON.stringify

File: platform_events/index.js [hunk 8]
```diff
@@ -410,11 +406,13 @@ module.exports.handler = (event, context, cb) => {
...
 						cb(JSON.stringify(errorHandler.throwInputValidationError("Bad request. message: " + err.message)));
 					}
 
 				} else {
-					logger.info(results.store_context);
+					
+					logger.verbose('POST success' + results.store_context)
```
[the remark this answers is not in the document]
Comment (supritat) [reply platform_events/index.js]: removed

File: platform_events/test/test.js [hunk 9]
```diff
@@ -16,14 +16,502 @@
...
+        AWS.restore(serviceName);
+        var bool = sinonSpy.called;
+        return bool;
+    };
+
+    beforeEach(function(){
+        spy = sinon.spy();
+        event = {
+            "method" : "",
+            "stage" : "test",
```
Comment (suryajak) [inline platform_events/test/test.js]: Most of the tests are failing as you don't have a respective config file for stage "test". Curious, did it pass for you locally?

Comment (supritat) [reply platform_events/test/test.js]: Included test-config.json to resolve this issue

**Pull request merged**